tests/generators/test_random_generator.py

```diff
@@ -59,3 +59,29 @@ def test_randomgen_config(self):
         npt.assert_equal(gen.lb.numpy(), np.array(lb))
         npt.assert_equal(gen.ub.numpy(), np.array(ub))
         self.assertEqual(gen.dim, len(lb))
+
+    def test_axrandom_config(self):
+        config_str = """
+                [common]
+                parnames = [par1, par2]
+                lb = [-1, 0]
+                ub = [1, 2]
+                outcome_types = [continuous]
+                strategy_names = [init]
+
+                [init]
+                generator = RandomGenerator
+                [RandomGenerator]
+                seed=231
+                deduplicate=True
+                """
+        config = Config(config_str=config_str)
+        gen = AxRandomGenerator.from_config(config, name="init")
+        self.assertEqual(gen.model, Models.UNIFORM)
+        self.assertEqual(gen.model_kwargs["seed"], 231)
+        self.assertTrue(gen.model_kwargs["deduplicate"])
+
```

Comment:
Here I would also test that the generator obeys parameter bounds (this can often catch shape and scale errors).